index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Youtube page</title>
+    <link rel="stylesheet" href="Styles/stylessheet.css" />
+    <link rel="shortcut icon" href="assets/image/youtubeLogo.png" />
+  </head>
+  <body>
+    <nav class="navbar whiteBGColor">
+      <ul class="displayFlexbox">
+        <li class="leftcornercontiner displayFlexbox">
+          <button class="Hamburger"></button>
+          <img
+            src="assets/image/youtubeLogoText.svg"
+            alt="yotube logo"
+            class="logotop"
+            srcset=""
+          />
+        </li>
+        <li>
+          <form action="" class="displayFlexbox">
+            <input type="text" name="" class="searchbar" placeholder="Search" />
+            <button class="submitbutton" type="submit"></button>
+          </form>
+        </li>
+        <li class="UploadAndSignContainer displayFlexbox">
+          <button class="upload"></button>
+          <button class="SignIn">Sign in</button>
+        </li>
+      </ul>
+    </nav>
+    <header class="popUp displayFlexbox whiteBGColor">
+      <h4 class="displayFlexbox">
+        <!--  <img src="" alt="" />-->
```

Comment:
Remove unused comment

Reply:
Thanks, we added more changes and created more icons for the left side of the navbar.